prow/spyglass/lenses/junit/lens.go

```diff
@@ -88,15 +97,34 @@ type TestResult struct {
 
 // Body renders the <body> for JUnit tests
 func (lens Lens) Body(artifacts []lenses.Artifact, resourceDir string, data string, config json.RawMessage) string {
+	jvd := lens.getJvd(artifacts)
+
+	junitTemplate, err := template.ParseFiles(filepath.Join(resourceDir, "template.html"))
+	if err != nil {
+		logrus.WithError(err).Error("Error executing template.")
+		return fmt.Sprintf("Failed to load template file: %v", err)
+	}
+
+	var buf bytes.Buffer
+	if err := junitTemplate.ExecuteTemplate(&buf, "body", jvd); err != nil {
+		logrus.WithError(err).Error("Error executing template.")
+	}
+
+	return buf.String()
+}
+
+func (lens Lens) getJvd(artifacts []lenses.Artifact) JVD {
```

Comment:
We may want to allow people to toggle between the current and previous functionality via a flag.

Reply:
Sounds like it might be fine to just ship it: https://kubernetes.slack.com/archives/CDECRSC5U/p1585349532027400?thread_ts=1585348940.027100&cid=CDECRSC5U

Reply:
Great to know. Will update based on other comments